langs/gleam/gleam

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+[ "$1" = "version" ] && exec /usr/local/gleam/bin/gleam --version
+
+cd /gleam
+cat - > /gleam/src/main.gleam
+
+shift
+exec /usr/local/gleam/bin/gleam run -t javascript "$@"
```

Comment:
the path to gleam executable doesn't match where it was placed in the Dockerfile
```suggestion
[ "$1" = "version" ] && exec /usr/bin/gleam --version

cd /gleam
cat - > /gleam/src/main.gleam

shift
exec /usr/bin/gleam run -t javascript "$@"
```

Reply:
You may omit `/usr/bin/` in wrapper shells and just type the binary's name. In the event you are repeatedly forced to use long paths, define an `export` under the shebang to still type only the binary's name. Alternatively, as you'll need to define the wrapper shell in `config/data/langs.toml` as the entry point, you may opt to insert an `env` entry under the `args` entry, like so:

```gleam
[Gleam]
args = [ '/usr/bin/gleamwrapper', '-' ]
env  = [ 'NODE_ICU_DATA=/usr/share/icu', 'PATH=/usr/local/bin:$PATH' ]
...
```